src/storage/FCS.cpp

```diff
@@ -142,6 +142,16 @@ bool FB_Storage::mDownload(FirebaseData *fbdo, const char *bucketID, const char
     return sendRequest(fbdo, &req);
 }
 
+bool FB_Storage::mDownloadOTA(FirebaseData *fbdo, const char *bucketID, const char *remoteFileName)
+{
+    struct fb_esp_fcs_req_t req;
+    req.remoteFileName = remoteFileName;
+    req.storageType = mem_storage_type_ota_update;
+    req.requestType = fb_esp_fcs_request_type_download;

```

Comment:
I think we should add the new type of request instead of storage type by adding the new enum for fb_esp_fcs_request_type e.g. fb_esp_fcs_request_type_download_ota

https://github.com/mobizt/Firebase-ESP-Client/blob/6e017585b37c4a3cec6f9e0a92f3cc5acb13fa56/src/common.h#L336-L345

`req.requestType = fb_esp_fcs_request_type_download_ota;`